lib/implore.es6.js

```diff
@@ -60,8 +61,22 @@ export default function implore(request) {
 			'implore requires parameter `method` to be a string'
 		);
 
+		const {requestHeaders={}} = request;
+		invariant(
+			typeof requestHeaders === 'object',
+			'implore requires parameter `requestHeaders` to be an object'
+		);
+
```

Comment:
Not sure how serious this is, but this use of default assignment + `invariant` isn't quite enough to ensure that `Object.keys(requestHeaders)` is safe to use below. If someone decides to pass along a `request` object with `requestHeaders` set to `null`, we'll try to run `Object.keys` on null and an exception will be thrown. E.g.:

```
const myRequest = { requestHeaders: null };
const {requestHeaders={}} = request; // `requestHeaders` will be set to `null` here
invariant(
    typeof requestHeaders === 'object', // passes since typeof `null` is object :(
    'implore requires parameter `requestHeaders` to be an object'
);
/* . . . */
Object.keys(requestHeaders).forEach(() => {}); // throws an exception
```

We could check to prevent this as well, or assume that no one would set `requestHeaders` to null. I prefer the former. Thoughts?